package/MDAnalysis/analysis/psa.py

```diff
@@ -573,6 +573,10 @@ def dist_mat_to_vec(N, i, j):
     :Returns:
       int, index (of the matrix element) in the corresponding distance vector
     """
+    try:
+        validate_dist_mat_to_vec_inputs(N, i, j)
+    except Exception as E:
+        raise E
```

Comment:
If you don't want to catch the exception and to some special error handling with it then you don't need the try/except block. You can remove it here.